hapi-fhir-opensrp-security-config/pom.xml

```diff
@@ -0,0 +1,165 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+
+	<parent>
+		<groupId>ca.uhn.hapi.fhir</groupId>
+		<artifactId>hapi-deployable-pom</artifactId>
+		<version>5.4.0-PRE5-SNAPSHOT</version>
+		<relativePath>../../hapi-deployable-pom/pom.xml</relativePath>
+	</parent>
```

Comment:
I think that this parent causes the artefact to inherit the groupId `ca.uhn.hapi.fhir`. I also think that this module will be published to Sonatype. We would need to own this groupId `ca.uhn.hapi.fhir` to publish to it. Therefore, we need to change this to `org.smartregister`

Reply:
@ekigamba I agree on this.  Do we maintain the versioning too or should we reset it too?